executor/runner/kernel_logs.go

```diff
@@ -0,0 +1,118 @@
+package runner
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func readCgroupFile(pid int) ([]string, error) {
+	rawCgroups, err := os.Open(fmt.Sprintf("/proc/%d/cgroup", pid))
```

Comment:
We already have code like this, that parses cgroup files,
can you look at `setCgroupOwnership` to see if you can reuse?

Reply:
Not re-use directly, but I can commonize it. Will do that.